frontend/src/components/workspaces/DashboardLogic/DashboardLogic.tsx

```diff
@@ -1,25 +1,41 @@
 import { Spin } from 'antd';
-import { useContext } from 'react';
+import { FC, useContext, useEffect, useState } from 'react';
 import { AuthContext } from '../../../contexts/AuthContext';
-import { useTenantQuery } from '../../../generated-types';
+import { TenantQuery, useTenantQuery } from '../../../generated-types';
+
+import { updatedTenant } from '../../../graphql-components/subscription';
 import Dashboard from '../Dashboard/Dashboard';
 
-function DashboardLogic() {
+const DashboardLogic: FC<{}> = () => {
   const { userId } = useContext(AuthContext);
+  const [data, setData] = useState<TenantQuery>();
 
-  const { data, loading, error } = useTenantQuery({
+  const { loading, error, subscribeToMore } = useTenantQuery({
     variables: { tenantId: userId ?? '' },
     notifyOnNetworkStatusChange: true,
+    onCompleted: setData,
   });
 
-  //startPolling(20000);
+  useEffect(() => {
+    if (!loading) {
+      subscribeToMore({
+        variables: { tenantId: userId ?? '' },
+        document: updatedTenant,
+        updateQuery: (prev, { subscriptionData }) => {
+          if (!subscriptionData.data) return prev;
+          setData(subscriptionData.data);
+          return subscriptionData.data;
```

Comment:
I would also extract `data` (can be done in the lambda definition directly, something like `{ { data } : subscriptionData }`, I'm not really sure about the syntax order though)

Reply:
I did it